scissors/main.c

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+enum result {ERROR = -1, TIE, USR_LOSE, USR_WIN};
+
+enum result play(void)
+{
+	enum optn {ROCK, PAPER, SCISSORS};
+	enum optn usr_optn;
+	enum optn mchn_optn = rand()%3;
+	char usr_optn_input;
+	enum result res;
+
+	fflush(stdin);
+	printf("Please choose: 0 (rock) or 1 (paper) or 2 (scissors)\n");
+	scanf(" %c", &usr_optn_input);
+
+	if (usr_optn_input != '0' && usr_optn_input != '1'
+			&& usr_optn_input != '2') {
+		printf("Wrong input! Error\n");
+		return -1;
+	}
+
+	usr_optn = atoi(&usr_optn_input);
```

Comment:
Sorry, I didn't notice this at first.
`usr_optn_input` is a single character, not string. It works just thanks to luck.
Either convert `usr_optn_input` into two-character array with explicit 0 in the second element or use another way to convert into a number, say, `usr_optn = usr_optn_input - '0'`